doc/source/whatsnew/v1.4.4.rst

```diff
@@ -14,8 +14,7 @@ including other versions of pandas.
 
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
--
--
+- Fixed regression in :meth:`DataFrame.eval` called with ``inplace=True`` had no effect (:issue:`47449`)
```

Comment:
the `inplace` argument also affects the return type so can perhaps reword for clarity and to be consistent with the release note for #47327

```suggestion
- Fixed regression in :meth:`DataFrame.eval` creating a copy when updating inplace (:issue:`47449`)
```

Reply:
done